Dockerfile

```
@@ -0,0 +1,9 @@
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM --platform=$BUILDPLATFORM gcr.io/distroless/static:nonroot
+ARG TARGETPLATFORM
```

Suggest defaulting this arg to `linux/amd64` so it just works for a regular `docker build` invocation:
```suggestion
ARG TARGETPLATFORM=linux/amd64
```

---

If I run `make docker-build` as is, then `docker run` the image, I get `exec /usr/local/bin/envoy-gateway: no such file or directory`. If I add the default I suggested, then do the same, I get `Manages Envoy Proxy as a standalone or Kubernetes-based application gateway` as expected.

---

It is environment dependent, if you did `docker buildx install` then your `docker build` is buildx otherwise it's not.  Though defaults to linux/amd64 doesn't do right thing for M1 mac users whose docker is linux/arm64.

---

Actually, set environment DOCKER_BUILDKIT=1 in the Makefile should make thing work? @skriss

---

Yep, understood, was just trying to have this work OOTB for the most environments. Since you're against the default, let's document the `buildx` requirement in https://github.com/envoyproxy/gateway/blob/main/CONTRIBUTING.md.